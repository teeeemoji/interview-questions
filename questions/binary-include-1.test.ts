import binaryInclude1 from './binary-include-1'

test('11111111', () => {
    expect(binaryInclude1(parseInt('11111111', 2))).toMatchSnapshot()
})

test('10101010', () => {
    expect(binaryInclude1(parseInt('10101010', 2))).toMatchSnapshot()
})

test('10110000', () => {
    expect(binaryInclude1(parseInt('10110000', 2))).toMatchSnapshot()
})
 