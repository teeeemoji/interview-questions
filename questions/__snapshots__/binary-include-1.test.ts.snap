// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`10101010 1`] = `4`;

exports[`10110000 1`] = `3`;

exports[`11111111 1`] = `8`;
